src/GovUk.Education.ExploreEducationStatistics.Data.Model/Services/FilterItemService.cs

```diff
@@ -15,18 +14,11 @@ public class FilterItemService : AbstractRepository<FilterItem, long>, IFilterIt
         {
         }
 
-        public IEnumerable<FilterItem> GetFilterItems(IQueryable<Observation> observations)
-        {
-            return observations.SelectMany(observation => observation.FilterItems)
-                .Select(item => item.FilterItem).Distinct();
-        }
-
         public IEnumerable<FilterItem> GetFilterItemsIncludingFilters(IQueryable<Observation> observations)
         {
-            var filterItems = observations.SelectMany(observation => observation.FilterItems)
+            var filterItems = observations
+                .SelectMany(observation => observation.FilterItems)
                 .Select(item => item.FilterItem)
-                .Include(item => item.FilterGroup)
-                .ThenInclude(group => group.Filter)
```

Comment:
The name of this method is a bit misleading now, it doesn't appear to get the FilterItems including Filters anymore, unless a previous call has already fetched them in the same database context.